test/e2e/validation_test.go

```diff
@@ -60,6 +60,7 @@ func testNodeMetadata(t *testing.T) {
 	for _, node := range gc.allNodes() {
 		t.Run(node.GetName()+" Validation Tests", func(t *testing.T) {
 			t.Run("Kubelet Version", func(t *testing.T) {
+				t.Skip("Kubelet Version test disabled")
```

Comment:
I'd think this should just be removed and not skipped. There are no plans to re-enable this.

Reply:
Removed the test and the associated helper functions.